### ISSUE TITLE
[FR] Rectangular selection embeds can be editable, imbedded and icon adjusted

### ISSUE THREAD

**Issue by Loners-xq** [RyotaUshio/obsidian-pdf-plus#226]
### Describe your request

Dear Developer,
First of all, I salute you and thank you for your great contribution to such an excellent plugin.

But there are two small questions.
1. When I use ' 'Rectangular selection embeds'', I can only edit the box once, and when I want to readjust it, I can only delete the re-box selection. Ask if you can set it to a reconfigurable style and leave its reference links unchanged.

2. ' 'Rectangular selection embeds'' This annotation cannot be embedded in the PDF source file. Can it be added directly to the source file? It's like highlighting.

3. Can the icon in the left column be moved to the top of the pdf document, so that my mouse can click more easily and avoid large movement?

Thank you again for your great effort.

**Comment by RyotaUshio** [comment]
Thank you for the kind words!

1. This is a duplicate of #82.
2. Yeah, it would be useful. I'm planning to implement it after #82 is resolved.
3. This is a duplicate of #151.

**Comment by RyotaUshio** [comment]
Closing this since the problem 2 will be tracked in #382.
